packages/markdown-docx/src/ToOOXMLVisitor/helpers.js

```diff
@@ -35,14 +35,33 @@ function titleGenerator(title, type) {
     return `${title} | ${type}`;
 }
 
+/**
+ * Wraps the OOXML in locked content controls
```

Comment:
Is this adding any extra information to explain what the function is doing? No, this is already clear by the function name. I encourage you write something more descriptive.